asm/unk_020023C0.s

```diff
@@ -2690,109 +2690,109 @@ _020037A6:
 	ldr r0, [r4, #0x4]
 	ldr r2, [r4, #0x8]
 	mov r1, #0x0
-	blx FUN_020C9550
+	blx GX_LoadBGPltt_2
```

Comment:
Drop the `_2` extension from the library functions and put it instead on the routines that tail-call them.